src/BookForm.tsx

```diff
@@ -130,6 +137,7 @@ const BookFormik: React.FC<BookFormProps> = ({
       }}
       initialValues={book}
       onSubmit={(values: Book) => {
+        console.log(values);
```

Comment:
Not sure if it matters or not, but it may be best not to log

Reply:
Wope! An artifact from testing. I'll remove it.